docs/content/preview/yugabyte-platform/upgrade/upgrade-yp-airgapped.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Upgrade air-gapped YugabyteDB Anywhere installation
+headerTitle: Upgrade air-gapped YugabyteDB Anywhere installation
+linkTitle: Upgrade air-gapped installation
+description: Upgrade air-gapped YugabyteDB Anywhere installation
+menu:
+  preview_yugabyte-platform:
+    identifier: upgrade-yp-airgapped
+    parent: upgrade
+    weight: 81
+type: docs
+---
+
```

Comment:
Should these pages be on tabs like https://deploy-preview-15077--infallible-bardeen-164bc9.netlify.app/preview/yugabyte-platform/install-yugabyte-platform/install-software/default/

Reply:
I'm not sure. I'm planning to create a separate ticket for the tabs and look at every section.